packages/usa/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "country-data-usa",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
you should do the version bump on a separate commit, and update the changelog too